cmd/ketch/app_info.go

```diff
@@ -85,24 +86,30 @@ func appInfo(ctx context.Context, cfg config, options appInfoOptions, out io.Wri
 	if err := cfg.Client().Get(ctx, types.NamespacedName{Name: app.Spec.Pool}, pool); err != nil {
 		return fmt.Errorf("failed to get pool: %w", err)
 	}
+
 	buf := bytes.Buffer{}
 	t := template.Must(template.New("app-info").Parse(appInfoTemplate))
 	table := &bytes.Buffer{}
 	w := tabwriter.NewWriter(table, 0, 4, 4, ' ', 0)
-	fmt.Fprintln(w, "DEPLOYMENT VERSION\tIMAGE\tPROCESS NAME\t#UNITS\tCMD")
+	fmt.Fprintln(w, "DEPLOYMENT VERSION\tIMAGE\tPROCESS NAME\tSTATE\tCMD")
 	noProcesses := true
 	for _, deployment := range app.Spec.Deployments {
 		for _, process := range deployment.Processes {
 			noProcesses = false
-			units := fmt.Sprintf("%d", ketchv1.DefaultNumberOfUnits)
-			if process.Units != nil {
-				units = fmt.Sprintf("%d", *process.Units)
+
+			pods, err := cfg.KubernetesClient().CoreV1().Pods(app.Namespace).List(ctx, metav1.ListOptions{
+				LabelSelector: fmt.Sprintf(`theketch.io/app-name=%s,theketch.io/app-deployment-version=%s,theketch.io/app-process=%s`, app.Name, deployment.Version, process.Name),
```

Comment:
it would be helpful to put labels to "labels.go", 
could you take it from 
https://github.com/shipa-corp/ketch/pull/45/files#diff-59fb47386f112a9a44e9e7fcf9594bf3346a4afa9ae47a170a9c20d1e3752bc6R4?

also, this pattern of querying kubernetes doesn't scale well, 
can we get all pods of all ketch applications before these loops and group them by an app, process, and deployment?

Reply:
Well, had same idea initially, but we can't get all pods for all apps, because apps could be in different namespaces. So, we need to do it at least for each app. Then, if we already have to do so per each app - there's not much improvement with getting all pods per app, because, usually - there's just only one deployment per app, and one process. Sure, there could be more in some rare cases, but overall, code readability will be worse with this patch. It not worth it.

Reply:
https://github.com/datawire/ambassador/blob/master/pkg/k8s/client.go#L36